terraform.md

```diff
@@ -1,29 +1,83 @@
    - [Secret Manager API](https://console.cloud.google.com/apis/library/secretmanager.googleapis.com)
    - [Certificate Manager API](https://console.cloud.google.com/apis/library/certificatemanager.googleapis.com)
    - [Compute Engine API](https://console.cloud.google.com/apis/library/compute.googleapis.com)
    - [Artifact Registry API](https://console.cloud.google.com/apis/library/artifactregistry.googleapis.com)
    - [OS Config API](https://console.cloud.google.com/apis/library/osconfig.googleapis.com)
    - [Stackdriver Monitoring API](https://console.cloud.google.com/apis/library/monitoring.googleapis.com)
    - [Stackdriver Logging API](https://console.cloud.google.com/apis/library/logging.googleapis.com)
-6. You will need domain on cloudflare
-7. Run `make init`
-8. build cluster disk image [here](./packages/cluster-disk-image)
-9. Fill in cloudflare API key with access to your domain
-10. Run `make build-and-upload-all`
-11. Run `make apply-without-jobs`
-12. Fill in following secrets:
-    - postgres (required)
-    - If you don't want to use, keep as it is:
-      - posthog (if you don't want to use posthog)
-      - grafana (if you want traces / logging)
-13. Run `make apply`
-14. Provisioning of the certificates can take some time, you can check the status in the Google Cloud Console
+6. Run `make build-cluster-disk-image`
+7. Run `make build-and-upload-docker-images`
+8. Run `make build-and-upload-fc-components` Note: This needs to be done on a Linux machine due to case-sensitive requirements for the file system--you'll error out during the automated git section with a complaint about unsaved changes. Kernel and versions could alternatively be sourced elsewhere.
+9. At the time of this writing, several versions are required. The script may not fully create and upload these. As of 9/27/24, your Storage buckets should look like this:  
+```
+<prefix>-fc-env-pipeline/envd  
+                        /envd-v.0.0.1 (this is legacy)  
+                        /orchestrator  
+                        /template-manager
+
+<prefix>-fc-kernels/vmlinux-5.10.186/vmlinux.bin
+                   /vmlinux-6.1.99/vmlinux.bin
+
+<prefix>-fc-versions/v1.7.0-dev_8bb88311/firecracker
+                                        /uffd
+                    /v1.8.0-hugepages-state_4778a02/firecracker
+                    /v1.8.0-hugepages-state_9e0b47a/firecracker
+                    /v1.8.0-main_43ff620/firecracker
+                    /v1.9.0_fake-2476009/firecracker
+                                        /uffd
+
+```
+10. Run `make apply-without-jobs`
+11. Secrets are created and stored in GCP Secrets Manager. Once created, that is the source of truth--you will need to update values there to make changes.   
+Some notes:  
+- You can optionally add Grafana and Posthog API keys for monitoring
```

Comment:
postgres_connection_string is still required

Reply:
good call, added